.rubocop.yml

```diff
@@ -63,6 +72,15 @@ Metrics/LineLength:
 Metrics/ParameterLists:
   CountKeywordArgs: false
 
+Naming/AccessorMethodName:
+  Exclude:
+    - 'lib/reek/context/visibility_tracker.rb'
+
+# EOS is a fine name to use in our specs
```

Comment:
Just curious what the reason is. Asking because I use an editor that will infer which syntax highlight to use based on the delimiter. I guess this could potentially be undesirable if the fixture adds too much noise?

Reply:
In this case, 'fine' means that I don't particularly care about which delimiter is used. I actually have half a mind to disable this cop for the entire project.

I don't have any experience with editors that highlight based on the delimiter used, but I can imagine that it becomes rather confusing if the strings represent Ruby code and are highlighted as such.

Reply:
It mostly saves me when working with custom SQL queries in Rails apps, but I agree this is a minor convenience (or inconvenience depending on your preference.) 🙂